setup.py

```diff
@@ -39,7 +39,7 @@
     'packages' : ['jinjasql'],
     'test_suite' : 'tests.all_tests',
     'install_requires': [
-        'Jinja2>=2.5,<3.0'
+        'Jinja2>=2.8'
```

Comment:
Why is this 2.8, but it's 2.5 in requirements.txt?

Reply:
I've updated the requirements one as well.